installation/resources/installer-config-local.yaml.tpl

```diff
@@ -103,3 +91,15 @@ metadata:
 data:
   knative.ingressgateway.service.type: NodePort
   knative.domainName: "kyma.local"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: assetstore-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: assetstore
+data:
+  minio.resources.requests.memory: 64Mi
+  minio.resources.limits.cpu: 100m
```

Comment:
Shouldn't it be set in `values.yaml`?

Reply:
This is an override for local development - minikube. In `values.yaml` we have a production setup.